app/views/options-on-interstitial/create-new-account-1.html

```diff
@@ -38,19 +45,37 @@ <h2 class="heading-medium">Government Gateway</h2>
     <div class="column-two-thirds">
       <h2 class="heading-medium">GOV.UK Verify</h2>
       <p>
-        Lorem ipsum dolor sit amet, te vix nisl splendide signiferumque, mel reque appetere cu. Pro eu populo conceptam. Id copiosae dissentias eam. Eum ea dico exerci dicunt, mea te habeo.
+        Registering with GOV.UK Verify works best if you have a UK address, plus either:
       </p>
 
+      <ul class="list list-bullet">
+        <li>a valid passport</li>
+        <li>a photocard driving licence</li>
+      </ul>
+
       <div class="inline-group">
         <div class="inline-group--item">
-          <a class="button button--start" href="https://www.tax.service.gov.uk/check-your-state-pension/signin/verify" role="button">
-            Create a new GOV.UK Verify account
+          <a role="button" class="button button--start" href="https://www.signin.service.gov.uk/start">
```

Comment:
I think this needs to be `https://www.tax.service.gov.uk/check-your-state-pension/signin/verify` otherwise you get a timeout page.

Reply:
Looks like it works if you've already visited that page